Makefile

```diff
@@ -0,0 +1,19 @@
+IMAGE_NAME ?= phys_standalone
```

Comment:
Now that we have a Makefile it would make sense to remove the `build.sh` script and add an `enter` and `stop` target (instead of having the `enter.sh` script) to unify the way we work with this repository.